src/selectors/getAllProjectKeys.js

```diff
@@ -1,10 +1,25 @@
 import {createSelector} from 'reselect';
+import isNull from 'lodash-es/isNull';
 
 export default createSelector(
   state => state.get('projects'),
   projects => projects.
-    filter(project => project.updatedAt).
-    sortBy(project => -project.updatedAt).
+    sort((
+      {updatedAt: firstProjectUpdatedAt,
+        updatedAt: secondProjectUpdatedAt}) => {
```

Comment:
Hey dude—testing it out I noticed the sort looked off, and the destructuring here isn’t quite right. You want to destructure the two args as separate objects. (Actually wish the linter caught this as right now you’re destructuring the same property twice, which is clear lint). So:

```js
sort((
  {updatedAt: firstProjectUpdatedAt},
  {updatedAt: secondProjectUpdatedAt},
) => {
```

Reply:
Oops! First time destructurer here.